demo_usage/visualize_30hz_benchmark_data_on_map.py

```diff
@@ -327,7 +327,7 @@ def render_front_camera_on_axis(self, ax: plt.Axes, timestamp: int, log_id: str)
 
 
 def visualize_30hz_benchmark_data_on_map(args: Any) -> None:
-    """ """
+    """Function to render lidar, imagery, and map elements overlaid on both a bird's-eye-view and egoview."""
```

Comment:
Thanks for the updates Ben -- should actually be `"Function to render lidar, tracks, and map elements overlaid on a bird's-eye-view"`

Reply:
Resolved.